### PR TITLE
C23_WMDE_Desktop_DE_17

### PR DESCRIPTION
**Pull request by gbirke**
Ticket: https://phabricator.wikimedia.org/T352507

### CODE REVIEW

**Comment by gbirke** [comment]
I based this on the Desktop 16 branch, will rebase & adapt this when that branch is merged.

If the VAR banner wins, we'll have to discuss if we create a new Theme (basically a clone of Treedip with different colors and some slider style changes) or how we structure theme variants to keep backwards compatibility

**Comment by Abban** [comment]
To me this looks like it should be a new theme, based on the amount of overrides happening in the banner styles.

**Comment by Abban** [comment]
Can you add the Already Donated feature tests to the banner tests? I forgot to add them when I added it back to the banner.

```js
describe( 'Already Donated', () => {
	test.each( [
		[ 'expectShowsAlreadyDonatedModal' ],
		[ 'expectHidesAlreadyDonatedModal' ],
		[ 'expectFiresMaybeLaterEvent' ],
		[ 'expectFiresGoAwayEvent' ]
	] )( '%s', async ( testName: string ) => {
		await alreadyDonatedModalFeatures[ testName ]( getWrapper() );
	} );
} );
```

File: test/banners/desktop/components/BannerVar.spec.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { afterEach, beforeEach, describe, expect, it, test, vi } from 'vitest';
+import { afterEach, beforeEach, describe, test, vi } from 'vitest';
 import { mount, VueWrapper } from '@vue/test-utils';
-import Banner from '../../../../banners/desktop/components/BannerVar.vue';
+import Banner from '../../../../banners/desktop/components/BannerCtrl.vue';
```
Comment (Abban) [inline test/banners/desktop/components/BannerVar.spec.ts]: This should import BannerVar

File: test/banners/desktop/components/BannerVar.spec.ts [hunk 2]
```diff
@@ -19,12 +20,12 @@ import { useFormModel } from '@src/components/composables/useFormModel';
...
 import { DynamicContent } from '@src/utils/DynamicContent/DynamicContent';
 import { bannerMainFeatures } from '@test/features/MainBanner';
-import { donorHeartFeatures, testDonorHeartValues } from '@test/features/DonorHeart';
+import { alreadyDonatedModalFeatures } from '@test/features/AlreadyDonatedModal';
 
 const formModel = useFormModel();
 const translator = ( key: string ): string => key;
 
-describe( 'BannerVar.vue', () => {
+describe( 'BannerCtrl.vue', () => {
```
Comment (Abban) [inline test/banners/desktop/components/BannerVar.spec.ts]: This should be BannerVar

**Review by Abban — APPROVED** [review]
Looks good, there's just a little clean up to do before merging.

There are some files you should delete:

- `visitorsVsDailyDonorsSentence.ts`
- `styles/MainBanner_var.scss`

Your IDE is adding 8 space indents to HTML, you can fix that in PhpStorm by going to `Settings > Editor > Code Style > HTML` and making sure all indents are set to 4

**Pull request merged**